### DISCUSSION TITLE
Switch to battery info app when charging

### DISCUSSION THREAD

**Discussion by Kozova1** [InfiniTimeOrg/InfiniTime#1611] [Ideas]
The Pinetime Lite fork has this feature where it switches to the battery app when charging.  
This could be a very nice feature.

**Comment by pfeerick** [comment]
Yes please! I was going to suggest and loot at writing a simple vibrate when put on charge or taken off charge, i.e. like the pebble, but I think this is a much better idea!

**Comment by qbinary** [comment]
I would propose to just show a popup like when a notification is received. But to make that look good the line should fill faster. I made this #306 issue to address that.

**Comment by moriel5** [comment]
I personally prefer to see the battery percentage at all times on the home screen, in a similar fashion to how LineageOS can show it on Android (number inside circular line that how "complete" it is also signifies the percentage, with an additional icon doing when it is charging).

**Comment by Riksu9000** [comment]
Would this address some problem, or what's the purpose of this?
